monitor_packages/mysql/mysql.json

```diff
@@ -0,0 +1,443 @@
+{
+  "name": "MySQL",
+  "description": "Folder for MySQL Monitors",
+  "type": "MonitorsLibraryFolderExport",
+  "children": [
+    {
+      "name": "MySQL - Large number of slow queries",
+      "description": "This alert fires when we detect that there are 5 or more slow queries within a 5 minute time interval.",
+      "type": "MonitorsLibraryMonitorExport",
+      "monitorType": "Metrics",
+      "queries": [
+        {
+          "rowId": "A",
+          "query": "$$mysql_data_source metric=mysql_slow_queries db_cluster=* server=* | sum by db_cluster, server"
```

Comment:
Make sure to mention replacing the placeholder value in docs, otherwise, monitors will fail.

Reply:
Doc contains section detailing instruction to perform find and replace with json import based monitors.